languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35467,6 +35467,21 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+      <rule id="SE_DEMANDER_QU_EST_CE_QUE" name="se demander qu’est-ce que" type="style" default="temp_off">
+          <pattern>
+              <token postag="R pers obj.*" postag_regexp="yes"/>
+              <token inflected="yes">demander</token>
+              <marker>
+                  <token>qu'</token>
+                  <token>est</token>
+                  <token>-ce</token>
+                  <token regexp="yes">qu.*</token>
+              </marker>
+          </pattern>
+          <message>Cette expression est familière. Souhaitez-vous écrire les termes <suggestion>ce que</suggestion> employés dans le language courant.</message>
```

Comment:
Nice rule! We should think about how to broaden it, I think it could catch more cases.

In the message, we should explain: "Dans le langage standard, cette question indirecte devrait être introduite par 'ce que'." ...or something like that.